docs/widgets.md

```diff
@@ -422,6 +427,7 @@ horizontal space as possible to fit the text.
 | `width` | `int` | Limits width of the area on which text may be drawn | N |
 | `linespacing` | `int` | Controls spacing between lines | N |
 | `color` | `color` | Desired font color | N |
+| `alignment` | `str` | Text Alignment | N |
```

Comment:
Would `align` be more in keeping with existing widgets? They use names like `main_align` and `cross_align` rather than `main_alignment` or `cross_alignment`

Reply:
Yeah I thought about it but had already submitted the PR , etc. I'll change it real quick.